### DISCUSSION TITLE
Hold, Release Hold?

### DISCUSSION THREAD

**Discussion by raeldor** [cubewise-code/tm1py#952] [General]
Hi,

I searched the documentation, but I couldn't see any reference to placing a 'Hold' or releasing a 'Hold' on the data.  Is it called something else?

Thanks
Ray

**Comment by MariusWirtz** [comment]
It's not implemented in TM1py yet. 
Please open an issue. I think this is something we should include in TM1py.

We might need to resort to TI to implement this feature though 🤔
https://www.tm1forum.com/viewtopic.php?p=82669#p82669

**Comment by wimgielis** [comment]
I added an example cube and TI process to implement holds based on a text
file. Could be of use when getting it set up in tm1py.
https://www.tm1forum.com/viewtopic.php?p=82682#p82682



Op ma 7 aug. 2023 om 10:39 schreef Marius Wirtz ***@***.***>

> It's not implemented in TM1py yet.
> Please open an issue. I think this is something we should include in TM1py.
>
> We might need to resort to TI to implement this feature though 🤔
> https://www.tm1forum.com/viewtopic.php?p=82669#p82669
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cubewise-code/tm1py/discussions/952#discussioncomment-6656200>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEDHULPKBSSSV3ETYPS7FC3XUCSURANCNFSM6AAAAAA3DX4HSU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 

------

Best regards / Beste groeten,

Wim Gielis
MS Excel MVP 2011-2014
https://www.wimgielis.com <http://www.wimgielis.be>
